diff --git a/Makefile b/Makefile
index e7d2f63..3695a84 100644
--- a/Makefile
+++ b/Makefile
@@ -22,7 +22,7 @@ PROGRAM = superiotool
 
 CC      = gcc
 INSTALL = /usr/bin/install
-PREFIX  = /usr/local
+PREFIX  = /usr
 
 # Set the superiotool version string from the highest revision number
 # of the checked out superiotool files.
@@ -31,7 +31,6 @@ SVNDEF := -D'SUPERIOTOOL_VERSION="$(shell svnversion -cn . \
 
 CFLAGS += -O2 -Wall -Wstrict-prototypes -Wundef -Wstrict-aliasing \
          -Werror-implicit-function-declaration -ansi -pedantic $(SVNDEF)
-LDFLAGS += -lz
 
 OBJS = superiotool.o serverengines.o ali.o fintek.o ite.o nsc.o nuvoton.o \
        smsc.o winbond.o infineon.o
@@ -43,9 +42,10 @@ endif
 ifeq ($(OS_ARCH), FreeBSD)
 CFLAGS = -O2 -Wall -Werror -Wstrict-prototypes -Wundef -Wstrict-aliasing \
          -Werror-implicit-function-declaration -ansi $(SVNDEF) \
-         -I/usr/local/include
-LDFLAGS += -L/usr/local/lib
-LIBS = -lz
+         -I/usr/include
+LDFLAGS += -L/usr/lib
+Libs.private: -lz
+LIBS = -L${libdir}
 endif
 ifeq ($(OS_ARCH), NetBSD)
 LDFLAGS = -l$(shell uname -p)
